### PR TITLE
Market statuses, market details view, additional seed data, and performance fixes for positions view

### PR DESCRIPTION
**Pull request by mikec**
still a few things left to add, but the chart is working now so figured I'd PR

### CODE REVIEW

**Comment by ehunter** [comment]
👍 looks good to me!

File: app/middleware/appEventInterceptor.js [hunk 1]
```diff
@@ -58,21 +62,25 @@ const appEventInterceptor = store => next => action => {
...
-          balances: [
-            total.yesShortBalance,
-            total.yesLongBalance,
-            total.noShortBalance,
-            total.noLongBalance
-          ]
-        }))
-      }
+
+      // TEMPORARILY REMOVING BECAUSE IT TAKES TOO LONG TO LOAD THIS DATA AND BLOCKS
```
Comment (jpkcambridge) [inline app/middleware/appEventInterceptor.js]: 😳

File: app/reducers/computed/tradePriceHistory.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      price: avgPrices.noPrice
+    })
+
+    timePointer += increment
+  }
+
+  return returnValues
+}
+
+function getAveragePricesOverRange(trades, start, end) {
```
Comment (jpkcambridge) [inline app/reducers/computed/tradePriceHistory.js]: This seems to be the average of all the marginal prices, not a time weighted average of the price.  I am not sure yet where it's being used, but wanted to double check that that's what we're looking for?

File: app/reducers/decisionMarkets.test.js [hunk 3]
```diff
@@ -143,21 +135,114 @@ describe('decisionMarkets', () => {
...
+            priceResolutionDate: '1500000003'
+          }
+        },
+        expected: [
+          { index: 0, props: { status: 'OPEN' } }
+        ]
+      },
+      {
+        when: 'when the decision action blocktime is greater than decisionResolutionDate but less than priceResolutionDate',
+        should: 'should set status to RESOLVEd',
```
Comment (jpkcambridge) [inline app/reducers/decisionMarkets.test.js]: lower case 'd'

**Comment by jpkcambridge** [comment]
One sanity check on averages.  Otherwise, looks good to me 👍

**Comment by mikec** [comment]
ready for re-review

**Comment by emaG3m** [comment]
The PR description says "Market Details" and it's 5000 lines of code changed lol. Might be good for our records to give the PR more description

**Comment by mikec** [comment]
@emaG3m fixed

File: app/views/DecisionDetail.js [hunk 4]
```diff
@@ -1,12 +1,81 @@
...
+  margin-left: 20px;
+`
+
+const ChartOuter = styled.div`
+  width: 360px;
+`
+
+const CircleChartSection = styled.div`
+
+`
```
Comment (ehunter) [inline app/views/DecisionDetail.js]: probably don't need this style?

**Comment by ehunter** [comment]
Just a minor comment about an empty style. Looks good to me 👍

**Comment by emaG3m** [comment]
cool. the description seems like now a jumble of unrelated things lol. Not sure what the purpose of this PR was?? did you add a graph or something? Will pull and look. Looking forward to shorter PR's ^_^

File: app/reducers/decisionMarkets.js [hunk 5]
```diff
@@ -65,11 +72,61 @@ const decisionMarkets = (state = [], action) => {
...
+  const decisionResolutionDateInt = parseInt(decisionResolutionDate)
+  const priceResolutionDateInt = parseInt(priceResolutionDate)
+  if (blocktimeInt < decisionResolutionDateInt)
+  {
+    return decisionStatuses.OPEN
+  } else if (blocktimeInt >= decisionResolutionDateInt && 
+             blocktimeInt < priceResolutionDateInt)
+  {
+    return decisionStatuses.RESOLVED
+  } else if (blocktimeInt >= priceResolutionDate) {
```
Comment (emaG3m) [inline app/reducers/decisionMarkets.js]: I think we should change this logic eventually

Comment (mikec) [reply app/reducers/decisionMarkets.js]: @emaG3m you're right .. maybe `CLOSED` is actually 2 statuses, `CLOSE_PENDING` and `CLOSE_COMPLETE` or something. we'll have to figure out how we'll handle on the UI

**Comment by emaG3m** [comment]
👍

**Pull request merged**